test/partial.js

```diff
@@ -21,4 +21,9 @@ describe('partial', function() {
     eq(g.length, 1);
   });
 
+  it('correctly processes rest arguments', function() {
+    var foo = (a, b, c, d, ...rest) => ({ a, b, c, d, rest });
+    var f = R.partial(foo, [100, 200]);
+    eq(f(1, 2, 3, 4), { a: 100, b: 200, c: 1, d: 2, rest: [3, 4] });
+  });
```

Comment:
So long as we're supporting ES5, I don't think we can include this test.

Reply:
So how should I modify the test, should these test files stay as before?

Reply:
I think we should remove the tests for `...rest` and add one for the version of `greet` that inspired #2940.  That should show the behavior well enough.

Reply:
Like this?:

```
const greet = (salutation, title, firstName, lastName) =>
  salutation + ', ' + title + ' ' + firstName + ' ' + lastName + '!';

const greetMsJaneJones = R.partialRight(greet, ['Ms.', 'Jane', 'Jones']);

greetMsJaneJones('Hello', 'Mr.' 'Green'); //=> 'Hello, Ms. Jane Jones!'
```

Reply:
exactly.

Reply:
Has changed the test as you suggested.